tests/acceptance/features/apiOcm/createInvitation.feature

```diff
@@ -31,33 +31,6 @@ Feature: create invitation
         }
       }
       """
-    When "Alice" lists the created invitations
```

Comment:
Is the deleted step/scenario covered or do we need to have separate scenario to list the created invitations?
cc @saw-jan

Reply:
@prashant-gurung899 we should mention what you are refactoring in this PR and why it is needed in the description, so elaborate on it. So that reviewer doesn't get confused and ask @saw-jan for everything.

Reply:
> @prashant-gurung899 we should mention what we are refactoring in this PR and why it is needed in the description. So that reviewer doesn't get confused and ask @saw-jan for everything.

Sure, we are refactoring multiple `When` steps where possible and aligning with the scenario title.
In the above scenario, we are just creating invitation as per the title. And yes it was discussed with @saw-jan to remove the `listing created invitations` step.

I will also update the PR description ASAP :+1:

Reply:
> Is the deleted step/scenario covered or do we need to have separate scenario to list the created invitations? cc @saw-jan

I think we need a separate test to list the created invitations as it hasn't been covered anywhere.

Reply:
It is better to have tests for deleted steps in the same PR.

Reply:
> It is better to have tests for deleted steps in the same PR.

Alright, i will write test to list the created invitation in the same PR. :+1:

Reply:
> > @prashant-gurung899 we should mention what we are refactoring in this PR and why it is needed in the description. So that reviewer doesn't get confused and ask @saw-jan for everything.
> 
> Sure, we are refactoring multiple `When` steps where possible and aligning with the scenario title. In the above scenario, we are just creating invitation as per the title. And yes it was discussed with @saw-jan to remove the `listing created invitations` step.
> 
> I will also update the PR description ASAP 👍 Also see: [#10264 (comment)](https://github.com/owncloud/ocis/pull/10264#issuecomment-2428602642)

i have pinged comment in description for better highlight

Reply:
Thank you